### ISSUE TITLE
UWP app with AutoMapper 5.0.0 and .NET Core 1.0 libraries won't build

### ISSUE THREAD

**Issue by DaveAurionix** [AutoMapper/AutoMapper#1472]
AutoMapper 5.0.0 works with UWP 5.0.0 but not with UWP 5.1.0 or UWP 5.2.0 (required for .NET Core 1.0).

Using .NET Core 1.0 class libraries in a Universal Windows Platform app requires Microsoft.NETCore.UniversalWindowsPlatform version 5.2.0.  Adding AutoMapper 5.0.0 produces the following build error:

_System.Reflection.Emit.ILGeneration 4.0.1 provides a compile-time reference assembly for System.Reflection.Emit.ILGeneration on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-x86-aot._

Example project.json to reproduce the problem in a blank new UWP app in Visual Studio 2015 with .NET Core 1.0 tools installed:

```
{
  "dependencies": {
    "AutoMapper": "5.0.0",
    "Microsoft.Extensions.Caching.Memory": "1.0.0",
    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.0"
  },
  "frameworks": {
    "uap10.0": {}
  },
  "runtimes": {
    "win10-arm": {},
    "win10-arm-aot": {},
    "win10-x86": {},
    "win10-x86-aot": {},
    "win10-x64": {},
    "win10-x64-aot": {}
  }
}

```
Removing AutoMapper resolves the build error, as does removing the example .NET Core class library (..Caching.Memory) and then downgrading UWP to 5.0.0.

**Comment by jbogard** [comment]
Duplicate, fixed in 5.0.1

**Comment by DaveAurionix** [comment]
Fantastic, thank you!  Sorry I was only seeing 5.0.0 as latest **stable** nuget package.  Can now see 5.0.2 and am consuming it successfully.
